parsers/grammars/YAML.g4

```diff
@@ -0,0 +1,1592 @@
+
+grammar Yaml;
+
+// Author: Syed Umar Anis
+
+tokens { INDENT, DEDENT }
+
+@lexer::members {
```

Comment:
I'm surprised to see Java code embedded here. As far as I know that should not be necessary. Did you test the grammar in any way?

Reply:
I haven't tested it yet cause I can't run the project on Windows. I am installing WSL and Install ANTLR4 in it to run the project, but I can't download ANTLR4 in the ubuntu yet, it's giving me some issue.